test_settings.py

```diff
@@ -13,7 +13,6 @@
     'csrf.apps.CsrfAppConfig',
     'django.contrib.auth',
     'django.contrib.contenttypes',
-    'django_nose',
```

Comment:
You should remove the `TEST_RUNNER` setting while you're at it, since it references a class in this package.